options.go

```diff
@@ -109,6 +109,7 @@ type BenchmarkOptions struct {
 	// Benchmark metrics can optionally be reported via statsd.
 	StatsdHostPort string `long:"statsd" description:"Optional host:port of a StatsD server to report metrics"`
 	PerPeerStats   bool   `long:"per-peer-stats" description:"Whether to emit stats by peer rather than aggregated"`
+	Format         string `long:"format" description:"Prints benchmark output in either plaintext or JSON format. Default is plaintext."`
```

Comment:
Couple of suggestions here:
 - we should use the `choice:` struct tag so the help page lists all the allowed options
 - we can use `default:` struct tag to indicate the default rather than calling it out in the description
 - we should probably support "text" or "plaintext" or something in addition to empty string to mean the default plaintext outpu.

something like:
```
Format string `long:"format" choice:"text" choice:"json" default:"text" description:"Output format for benchmark results"`
```

We also should update the test to validate that `"text"` works as well as `""`.

Once you make these changes, test that the `--help` output looks good.